alerts/absent_alerts.libsonnet

```diff
@@ -38,6 +38,27 @@
           },
         ],
       },
+      {
+        name: 'ceph-mds-status',
+        rules: [
+          {
+            alert: 'CephMdsIsAbsent',
+            expr: |||
+              absent(ceph_mds_metadata{%(cephExporterSelector)s, ceph_daemon=~"mds(.*)"} == 1)
+            ||| % $._config,
+            'for': $._config.mdsIsAbsentAlertTime,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Storage metadata service not available anymore.',
```

Comment:
Message and Description are the same. Would be good to have more info in description or discard it.

Reply:
Ack. Also as per comments from Paul, we will rather depend on count(ceph_mds_metadata) to alert if required replicas are available for MDS server.

Reply:
Ack. Also as per comments from Paul, we will rather depend on count(ceph_mds_metadata) to alert if required replicas are available for MDS server.